node/node_test.go

```diff
@@ -1149,3 +1090,57 @@ func prepChannel(t *testing.T, from, to address.Address, c, p *filecoin.MockLotu
 
 	return chAddr, collect
 }
+
+func MockFilBalance(t *testing.T, from, to address.Address, c *filecoin.MockLotusAPI, createAmt big.Int) {
```

Comment:
You can remove all this and just do:
```suggestion
c.SetActorState(&filecoin.ActorState{
		Balance: createAmt,
	})
```
The wallet method doesn't care about the rest.